design/vela-core/APIServer-Catalog.md

```diff
@@ -233,33 +246,11 @@ A catalog contains multiple packages, and a package contains multiple versions.
   url: "https://..."
   label:
     category: foo
-  # use existing Helm Chart as the manifest source for this package
-  chart:
-    # choose either local path or remote repo
-    path: ./chart/
-    remote:
-      repo: https://charts.bitnami.com/bitnami
-      name: redis
   ```
 
-  Note that we choose to integrate with existing community solutions instead of inventing another packaging solution (e.g. Helm, Terraform). In this way we can adopt the reservoir of community resources and also be extensible to more packaing formats.
-
-- Each package could expose multiple workload and trait definitions under `definitions/`. These definitions will be parsed on APIServer side to check if a cluster needs install or upgrade this package by comparing definitions.
+- `definitions`: definition files that describe the capabilities from this package to enable on a cluster. Note that these definitions will compared against a cluster on APIServer side to see if a cluster can install or upgrade this package.
 
-- The parameters of this package are defined in `parameters.yaml` . It will be used as the values to render the manifest source (e.g. Helm Chart).
-
```

Comment:
why parameters are deleted?

Reply:
This is not useful, and we shouldn't require users to write this file.
Because we can just parse the schema of parameters from Helm Chart or Terraform. For example, Helm supports [value schema file](https://www.arthurkoziel.com/validate-helm-chart-values-with-json-schemas/) for input validation and there is an [automation tool](https://github.com/karuppiah7890/helm-schema-gen] to generate the schema.